### ISSUE TITLE
Support for grouping rules

### ISSUE THREAD

**Issue by mkjensen** [societe-generale/arch-unit-build-plugin-core#70]
## Summary

Please consider supporting that a configurable rule can be a [grouping of rules](https://www.archunit.org/userguide/html/000_Index.html#_grouping_rules).

## Type of Issue

It is a : 
 - [ ] bug
 - [x] request
 - [ ] question regarding the documentation

## Motivation

Basically, given
```
class AllRulesTest {

    @ArchTest
    static final ArchTests SOME_RULES = ArchTests.in(SomeRules.class);

    @ArchTest
    static final ArchTests OTHER_RULES = ArchTests.in(OtherRules.class);
}
```
it would be nice that the Maven plugin would pick up all the rules via
```
(...)
<rules>
  <configurableRules>
      <configurableRule>
          <rule>package.AllRulesTest</rule>
      </configurableRule>
  </configurableRules>
</rules>
(...)
```
instead of having to do
```
(...)
<rules>
  <configurableRules>
      <configurableRule>
          <rule>package.SomeRules</rule>
      </configurableRule>
      <configurableRule>
          <rule>package.OtherRules</rule>
      </configurableRule>
  </configurableRules>
</rules>
(...)
```

## Current Behavior

~~~
Grouping is ignored (rules referred to by ArchTests are not taken into consideration).
~~~

Currently it looks like grouping is not supported in [InvokableRules](https://github.com/societe-generale/arch-unit-build-plugin-core/blob/a11ca988602f02f839d6a0ce181f229cf449db7d/src/main/java/com/societegenerale/commons/plugin/service/InvokableRules.java#L94).

## Expected Behavior

~~~
Grouping works via the plugin just as if run as a test directly.
~~~

## Steps to Reproduce (for bugs)

If the above is not clear, I can provide a sample.

## Your Environment

ArchUnit 1.0.1
arch-unit-maven-plugin 3.0.0

**Comment by vincent-fuchs** [comment]
the `configurableRule`s are supposed to be configured in the build config (maven or gradle). grouping would defeat the purpose, wouldn't it ? Maybe it could work with `preconfiguredRules` , but I am not convinced we would gain much.

**Comment by mkjensen** [comment]
You are right that one can continue to add rules to the plugin configuration.

The idea with this request is that the plugin configuration could be static and you could extend the referenced grouping class(es) instead.

The thinking is that keeping the plugin configuration static and leaving details to Java classes would be easier to maintain, but maybe this is subjective.

**Comment by mkjensen** [comment]
I added a PR (https://github.com/societe-generale/arch-unit-build-plugin-core/pull/95) that demonstrates a way to do this.
